tests/rptest/scale_tests/many_clients_test.py

```diff
@@ -64,17 +62,19 @@ def test_many_clients(self):
         RECORDS_PER_PRODUCER = 1000
 
         # Realistic conditions: 128MB is the segment size in the cloud
-        segment_size = 128 * 1024 * 1024
-        retention_size = 2 * segment_size
+        segment_size = 256 * 1024 * 1024
```

Comment:
What's the reason for doubling the segment size?

Reply:
ah I could probably just keep increasing the retention size (taking it 2x to 4x, maybe up to 6-8x).  Big issue was that logs would be cleaned up before they were consumed resulting in a mismatch between the amount we produced and the amount we consumed.

Reply:
Could we leave the segment size at 128MiB then and just increase retention to `8 * segment_size`? The 128MiB segment is somewhat important as it's what we use in our cloud offerings. Other than that it looks good to me.

Reply:
Ok I'll give that a shot